pages/articles/docker.md

```diff
@@ -0,0 +1,201 @@
+
+## Play with PnP.PowerShell online
+
+You can try using m365pnp/powershell Docker containers online, without installing anything on your desktop/laptop.
+
+1. Open https://labs.play-with-docker.com/
+
+2. Login
+
+3. Click `Start`
+
+4. Click `+ ADD NEW INSTANCE`
+
+5. Run in the online console:
+
+    ```
+    docker run --rm -it m365pnp/powershell
+    ```
+
+After that you can start running commands like `Connect-PnPOnline`.
+
+## Installing Docker locally
+
+\<to be verified\>
+
+- Mac OS:
+
+    1. Install
+
+    ```bash
+    brew install --cask docker
+    ```
+
+    2. Launch Docker:
+
+        - Press ⌘ + Space to bring up Spotlight Search and enter Docker to launch Docker.
+        - In the Docker needs privileged access dialog box, click OK.
+        - Enter password and click OK.
+
+- Linux:
+
+    ```bash
+    bash <(curl -s https://get.docker.com/)
+    sudo usermod -aG docker $USER
+    newgrp docker
+    ```
+
+- Linux containers on Windows (using WSL):
+
+    1. Start installing WSL:
+
+        ```powershell
+        Enable-WindowsOptionalFeature -Online -FeatureName "Microsoft-Windows-Subsystem-Linux", "VirtualMachinePlatform" -NoRestart
+        Invoke-WebRequest -Uri https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi -OutFile wsl_update_x64.msi -UseBasicParsing
+        Start-Process msiexec.exe -Wait -ArgumentList "/I $((Get-Location).Path)\wsl_update_x64.msi /quiet"
+        ```
+
+    2. Reboot the machine before you continue
+
+    3. Finish installing WSL:
+
+        ```powershell
+        wsl --set-default-version 2
+        wsl --install --distribution Ubuntu
+        ```
+
+    4. Install docker in WSL:
+
+        ```bash
+        bash <(curl -s https://get.docker.com/)
+        sudo usermod -aG docker $USER
+        newgrp docker
+        ```
+
+- Windows:
+
+    ```powershell
+    iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+    choco install -y docker-engine
+    ```
+
+## Using PnP.PowerShell
+
+Sometimes you want to run inline PnP.PowerShell commands.
+
+- Linux/WSL/Mac OS:
+
+    \<to be verified\>
+
+    ```bash
+    docker run --rm -it m365pnp/powershell
+    ```
+
+- Windows (run in PowerShell console):
+
+    ```powershell
+    docker run --rm -it m365pnp/powershell:1.10.0-nanoserver-1809
+    ```
+
+After that you can start running commands like `Connect-PnPOnline`.
+
+Mind you that in the case above, the container will have an isolated disk system so the commands that you run inside the container will not be able to access files from your host machine. However, sometimes you might want to run a script or use some files from your host OS (for example, when you have a ps1 file in a git repository that you cloned to your laptop). In this case you will need to mount a directory on your host as a volume inside the container. See the following examples.
+
+- Linux/WSL/Mac OS:
+
+    \<to be verified\>
+
```

Comment:
Here as well, can we please remove this section, the `to be verified` one ? If we are not sure, lets not add it and confuse users